### ISSUE TITLE
:bug: List Files Return Type

### ISSUE THREAD

**Issue by cdelcastillo21** [cdelcastillo21/taccjm#9]
TACCJobManager class `list_files` routine returns list of dictionaries with file info, but says it returns list of strings, which is what list_files in taccjm_server.py is expecting.

**Comment by cdelcastillo21** [comment]
Similar issue with list_job_files:

```
list_job_files error
---------------------------------------------------------------------------
TACCJMError                               Traceback (most recent call last)
Input In [105], in <module>
----> 1 tjm.list_job_files(jm, deployed['job_id'])

File ~/repos/taccjm/src/taccjm/taccjm.py:1017, in list_job_files(jm_id, job_id, path)
   1015     e.message = "list_job_files error"
   1016     logger.error(e.message)
-> 1017     raise e
   1019 return res

File ~/repos/taccjm/src/taccjm/taccjm.py:1012, in list_job_files(jm_id, job_id, path)
    984 """
    985 Get info on files in job directory.
    986 
   (...)
   1008         - asbytes  : Output from an ls -lat like command on file.
   1009 """
   1011 try:
-> 1012     res = api_call('GET',
   1013             f"{jm_id}/jobs/{job_id}/files/list", {'path': path})
   1014 except TACCJMError as e:
   1015     e.message = "list_job_files error"

File ~/repos/taccjm/src/taccjm/taccjm.py:177, in api_call(http_method, end_point, data)
    175     return json.loads(res.text)
    176 else:
--> 177     raise TACCJMError(res)

TACCJMError: <Response [500]>Unable to parse json errors. Raw content:A server error occurred.  Please contact the administrator.
```

In the log file for the server we see:

```
DEBUG:paramiko.transport:[chan 55] Sesch channel 55 request ok
INFO:paramiko.transport.sftp:[chan 55] Opened sftp connection (server version 3)
DEBUG:paramiko.transport.sftp:[chan 55] lstat(b'/scratch/06307/clos21/l1/jobs/test-job_20220203_162406/')
DEBUG:paramiko.transport.sftp:[chan 55] listdir(b'/scratch/06307/clos21/l1/jobs/test-job_20220203_162406/')
INFO:paramiko.transport.sftp:[chan 55] sftp session closed.
DEBUG:paramiko.transport:[chan 55] EOF sent (55)
Traceback (most recent call last):
  File "/Users/carlos/opt/miniconda3/envs/chg-prod/lib/python3.10/site-packages/hug/json_module.py", line 20, in __call__
    return self._dumps(*args, **kwargs)
TypeError: reject_bytes is on and '-rwx------   1 856065   800588        897 03 Feb 16:24 submit_script.sh' is bytes

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/carlos/opt/miniconda3/envs/chg-prod/lib/python3.10/wsgiref/handlers.py", line 137, in run
    self.result = application(self.environ, self.start_response)
  File "/Users/carlos/opt/miniconda3/envs/chg-prod/lib/python3.10/site-packages/falcon/api.py", line 269, in __call__
    responder(req, resp, **params)
  File "/Users/carlos/opt/miniconda3/envs/chg-prod/lib/python3.10/site-packages/hug/interface.py", line 947, in __call__
    raise exception
  File "/Users/carlos/opt/miniconda3/envs/chg-prod/lib/python3.10/site-packages/hug/interface.py", line 917, in __call__
    self.render_content(
  File "/Users/carlos/opt/miniconda3/envs/chg-prod/lib/python3.10/site-packages/hug/interface.py", line 853, in render_content
    content = self.outputs(
  File "/Users/carlos/opt/miniconda3/envs/chg-prod/lib/python3.10/site-packages/hug/output_format.py", line 174, in json
    return json_converter.dumps(
  File "/Users/carlos/opt/miniconda3/envs/chg-prod/lib/python3.10/site-packages/hug/json_module.py", line 22, in __call__
    raise TypeError("Type[ujson] is not Serializable", exc)
TypeError: ('Type[ujson] is not Serializable', TypeError("reject_bytes is on and '-rwx------   1 856065   800588        897 03 Feb 16:24 submit_script.sh' is bytes"))
127.0.0.1 - - [03/Feb/2022 16:24:53] "GET /l1/jobs/test-job_20220203_162406/files/list HTTP/1.1" 500 59
DEBUG:paramiko.transport:[chan 55] EOF received (55)
```

**Comment by cdelcastillo21** [comment]
Fixed in #13
